odkx-src/sync-endpoint-cloud-setup.rst

```diff
@@ -247,34 +247,37 @@ Setting up an Azure account
 Setting up a virtual machine
 """"""""""""""""""""""""""""
 
-1. First, click on the :guilabel:`Virtual Machines` button underneath the **Azure Services** section on the portal. Then, click on :guilabel:`Add` to create a new virtual machine.
+1. First, click on the :guilabel:`Virtual Machines` button underneath the **Azure Services** section on the portal. Then, click on :guilabel:`Create` to create a new virtual machine.
 
   .. image:: /img/setup-azure/azure1.png
    :width: 600
 
   .. image:: /img/setup-azure/azure2.png
    :width: 600
 
-2. Create a new resource group to attach to this virtual machine by clicking on :guilabel:`Create new`. Additionally, enter a name for the virtual machine and make sure that *Ubuntu Server 18.04 LTS* is selected for the image name.
-
   .. image:: /img/setup-azure/azure3.png
+   :width: 600
+
+2. Create a new resource group to attach to this virtual machine by clicking on :guilabel:`Create new`. Additionally, enter a name for the virtual machine and make sure that *Ubuntu Server 18.04 LTS* is selected for the image name as Ubuntu Server 20.04 LTS has not been tested yet.
+
+  .. image:: /img/setup-azure/azure4.png
     :width: 600
 
-3. Scroll down and select your authentication type. We highly recommend that use an SSH key for authentication. Copy and paste your SSH key username, and the key itself.
+3. Scroll down and select your authentication type. We highly recommend that use an SSH key for authentication. Copy and paste your SSH key username, and the key itself. Azure now provides an option to automatically generate an SSH key pair which can be stored for future use to connect to the virtual machine.
```

Comment:
I am not 100% sure what the added sentence means. Is this something people need to do? An option? When/why would they do it?